src/ocm/sentry.ts

```diff
@@ -16,3 +17,13 @@ export const captureException = (
       : console.warn(message, error);
   }
 };
+
+export const sentryErrorHandler: ErrorHandler = (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  error: any,
+  message?: string,
+  severity = 'error',
+) => {
+  const sentrySeverity = severity === 'error' ? Sentry.Severity.Error : Sentry.Severity.Warning;
+  return captureException(error, message, sentrySeverity);
+};
```

Comment:
This is just duplicating what `captureException` already does. What is this good for?

Reply:
Maybe `captureException` is not a good name and you'd like it to be called `sentryErrorHandling` instead? (If so, I'm with you on this)

Reply:
it's because of the mismatch in types. I'll find a different way to resolve this

Reply:
Sorry, I don't see what's the mismatch? Both functions seem to have the same signature

Reply:
the captureException uses the sentry error type, which can't be imported in common

Reply:
Then why not refactor `captureException` so it can receive a generic type for the severity argument?